README.md

```diff
@@ -33,7 +33,7 @@ The environment file output by `ilp-kit-config` can be used to start
 a ledger, UI, and connector. In the directory of ILP Kit, run:
 
 ```sh
-$ source ./config.sh ; npm start
+$ source ./env.list ; npm start
```

Comment:
I think sourcing the env.list actually doesn't work anymore. `npm start` should just load `env.list` automatically.